lib/find.ts

```diff
@@ -1,32 +1,42 @@
 import * as fs from 'fs';
 import { join } from 'path';
-import { promisify } from 'util';
 import { debug } from './debug';
 
-const readdir = promisify(fs.readdir);
-const stat = promisify(fs.stat);
+const MAX_FILE_SIZE: number = 2 * 1024 * 1024 * 1024;
```

Comment:
We could create a config.ts file where this size limit would be defined as exportable

`const MAX_FILE_SIZE: number = 2 * 1024 * 1024 * 1024;`

and have it reused here in find.ts and also in `find.test.ts` so you do not have to redefine it there.

Also what do you think of rename it from `MAX_FILE_SIZE` to `MAX_SUPPORTED_FILE_SIZE`?

Let me know what do you think :)

Reply:
I'm totally fine that it's defined somewhere else, it might even be better, but I guess config seems a bit misleading since it's not configurable.

Reply:
yes I agree, I think `common` would be more suitable for this use case. It was just me overthinking on later on support a --max-size flag 😄